### DISCUSSION TITLE
播放列表可以添加正序倒序的功能么？

### DISCUSSION THREAD

**Discussion by feipinxiang** [SPlayer-Dev/SPlayer#311] [想法 Ideas]
有些音频列表的内容需要倒序播放（比如一些按时间的播客内容），现在只能正序播放顺序不对。能添加倒序的功能么？

**Comment by feipinxiang** [comment]
是不是在src/components/Player/MainPlayList.vue加俩个排序按钮重排一下数组就可以了？

**Comment by imsyy** [comment]
![image](https://github.com/user-attachments/assets/d5d326de-b313-4bc8-8fad-5628bbf7f643)

不是有吗

**Reply by feipinxiang** [reply]
> 不是有吗

没有啊，我开的一个播客的播放列表，没看到啊。这个截图的位置是在哪里啊？
![image](https://github.com/user-attachments/assets/443707ed-f1ce-4f11-9831-fe5726310cc0)

![image](https://github.com/user-attachments/assets/a73ee262-1b87-4053-af9f-9353435f141b)

**Reply by feipinxiang** [reply]
看了SPlayer-2.0.9、3.0.0b1， 都没找到截图的那个排序设置啊

**Reply by imsyy** [reply]
> 看了SPlayer-2.0.9、3.0.0b1， 都没找到截图的那个排序设置啊

在歌曲列表的标题处点击

**Reply by feipinxiang** [reply]
> 在歌曲列表的标题处点击
在“播放队列”那四个字上右键么？方便截图说下在哪儿吗？我只能在这个位置右键出一个关闭菜单。我现在是3.0.0b1未登录状态。
![image](https://github.com/user-attachments/assets/24e7bc86-bb71-4bf9-8142-5ddd561f759d)

**Reply by feipinxiang** [reply]
找到了。是否能添加直接在播放队列这里排序呢？这边也能排的话会方便很多，不用再去找歌曲原位置操作了。

**Reply by feipinxiang** [reply]
又测试了下“标题”处的排序好像和播放队列的排序不是一回事。比如介绍页里按标题以日期排序后是1-50的顺序，点播放，到播放列表里还可能是50-1的顺序。（另外Electron的热键没关掉，即便播放器在后台按Ctrl+Shift+I也会出调试工具）

**Comment by feipinxiang** [comment]
又测试了下“标题”处的排序好像和播放队列的排序不是一回事。比如介绍页里按标题以日期排序后是1-50的顺序，点播放，到播放列表里还可能是50-1的顺序。（另外Electron的全局热键Ctrl+Shift+I没关掉，即便播放器在后台按Ctrl+Shift+I也会出调试工具）
